mlflow/genai/judges/instructions_judge/__init__.py

```diff
@@ -219,30 +219,29 @@ def __call__(
             # Build the user message with variable substitutions
             template_values = {}
             if inputs is not None and self._TEMPLATE_VARIABLE_INPUTS in self.template_variables:
-                template_values[self._TEMPLATE_VARIABLE_INPUTS] = json.dumps(
-                    inputs, default=str, indent=2
+                json_inputs = json.dumps(inputs, default=str, indent=2)
+                template_values[self._TEMPLATE_VARIABLE_INPUTS] = (
```

Comment:
@BenWilson2 does this affect just the substitution in the user prompt, so that each variable is "tagged" appropriately? If so then why don't we just change the user prompt and keep the logic here simpler?